packages/orchestrator-ui-components/src/components/WFOSteps/WFOStep/WFOStep.tsx

```diff
@@ -89,6 +98,16 @@ export const WFOStep = ({
                     )}
                 </EuiFlexGroup>
             </EuiFlexGroup>
+            {stepDetailIsOpen && (
+                <EuiCodeBlock
+                    isCopyable={true}
+                    language={'json'}
+                    lineNumbers={true}
+                    overflowHeight={6000}
+                >
+                    {JSON.stringify(delta, null, 3)}
```

Comment:
Maybe it is me, but I cannot stand a spacing of 3. Normally IDE's have either 2 or 4 configured as well. Personally I am team "4-spaces", but can live with 2 spaces as well.

Reply:
4 won't fit with large workflows (e.g. fw)
2 looks flimsy